CHANGELOG.md

```diff
@@ -15,8 +15,38 @@ and includes an additional section for migration notes.
 
 
 ## [Unreleased]
+### Changed
+- *ORCA-290* Renamed `excludeFileTypes`, `orcaDefaultBucketOverride` and `orcaDefaultStorageClassOverride` to `excludedFileExtensions`, `defaultBucketOverride` and  `defaultStorageClassOverride` respectively. In addition, ORCA configuration variables `excludedFileExtensions`, `defaultBucketOverride` and `defaultStorageClassOverride` are now under a new key `orca` under `meta` key.
```

Comment:
Switch to "are now under `collection.meta.orca` to include the full path and clarify ordering.

Reply:
updated